pkg/cli/cmd/radinit/aws.go

```diff
@@ -53,12 +54,24 @@ func (r *Runner) enterAWSCloudProvider(ctx context.Context) (*aws.Provider, erro
 		return nil, err
 	}
 
-	accountId, err := r.getAccountId(ctx, accessKeyID, secretAccessKey)
+	accountId, err := r.getAccountId(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	region, err := r.selectAWSRegion(ctx, QueryRegion, accessKeyID, secretAccessKey)
+	addingCloudProvider, err := prompt.YesOrNoPrompt(fmt.Sprintf(confirmAWSAccountIDPromptFmt, accountId), prompt.ConfirmYes, r.Prompter)
```

Comment:
this code is a bit difficult to understand, what is intended usage of `addingCloudProvider`? Isn't this decided based on the inputs for `confirmCloudProviderPrompt` and `selectCloudProviderPrompt` prompts?

Reply:
i missed to change the variable name here, updated it. changed it from `addingCloudProvider ` -> `addingAccountID`